articles/introduction-to-airflow/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/introduction-to-airflow/
+title: Introduction to Airflow
+description:
+author: adith-bharadwaj
+date: 2020-10-08T00:00:00-11:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-airflow/hero.jpg
+    alt: airflow example image
+---
+According to the [official documentation](https://airflow.apache.org/), *Airflow is a platform created by the community to programmatically author, schedule, and monitor workflows*.
+<!--more-->
+Airflow is a scalable and extensible platform that is rapidly gaining popularity in the data science community. In this article, we are going to explore the basics of Airflow and understand what makes it so popular.
+
+### What is Airflow?
+*Airflow is an **open-source** tool, written in [Python](https://www.python.org), for automating and scheduling tasks and workflows*. A workflow can be defined as a sequence of tasks that run on a specific schedule or get triggered by an event. Workflows are frequently used to handle big data processing [pipelines](https://www.alooma.com/blog/what-is-a-data-pipeline). A pipeline is a set of data processing elements connected in series. In a data pipeline, the output of one element is given as input to the next element in the series and so on. Airflow was created by Airbnb and donated to the Apache software foundation as an open-source project.
+
+
+A typical workflow looks like this:
+<ul>
+<li>Extract data from the source(database, API, etc.)</li>
+<li>Process the data by sending it somewhere</li>
+<li>Monitor the process</li>
+<li>Store the data in a data warehouse</li>
+<li>Transform the data and make it usable for analytics</li>
+<li>Analyze the data and send the report by email.</li>
+</ul>
+
+Airflow automates the process of monitoring, scheduling, and distributing these tasks in workflow across nodes. To accomplish this, Airflow provides a framework and a rich library to define these tasks in Python. It provides an aesthetic UI to view and monitor workflows in a web-based application and is extensible through plugins.
```

Comment:
> Airflow automates the process of monitoring, scheduling, and distributing these tasks in workflow across nodes.
 
It would be helpful to readers if you could define what a node is